build.gradle

```diff
@@ -119,3 +120,10 @@ compileJava {
 compileTestJava {
     options.compilerArgs += ["-Xlint:unchecked", "-Xlint:deprecation"]
 }
+
+startScripts {
+    doLast {
+        unixScript.text = unixScript.text
+            .replace('DEFAULT_JVM_OPTS=\"\"', 'DEFAULT_JVM_OPTS=\"-Xmx10g\"')
```

Comment:
what will this do on our laptops?  Like, if we're just running a FileSet request and have commented the rest out in WasapiDownloader ... would we need to comment this out as well?  (doable - just asking)